server/.gitignore

```diff
@@ -29,4 +29,7 @@ tarpaulin-report.html
 static/
 hashLocations.json
 
+config.json
+*.log
+Cargo.lock
```

Comment:
Remove -- we need this file, but only commit it on a release. (Or is there some nicer way to express "ignore this for now, but I need it later"?)